### DISCUSSION TITLE
how install depended modules in elyra

### DISCUSSION THREAD

**Discussion by pyshahid** [elyra-ai/elyra#2372] [Q&A]
what i want to know is if i created a pipeline component named test and in that test has to install a python module beautifulsoup4 how can i do that.

**Comment by ptitzler** [comment]
Can you clarify what your runtime environment is? For Kubeflow Pipelines, you'd have to install the dependencies in the component's container image. For Apache Airflow follow [these instructions](https://airflow.apache.org/docs/apache-airflow/stable/modules_management.html).
